src/util.cpp

```diff
@@ -592,26 +592,41 @@ fs::path GetConfigFile(const std::string& confPath)
     return pathConfigFile;
 }
 
+void ArgsManager::ReadConfigStream(fs::ifstream& streamConfig)
+{
+    std::set<std::string> setOptions;
```

Comment:
Is it possible to add a log to this function to output which file it's reading, or is it too early to start logging?

Sorry - by my earlier comment, I didn't mean you should remove the logging on failure, just that the structure of the function suggested to me that it shouldn't be called recursively.